### ISSUE TITLE
Home couldn't connect to this accessory.

### ISSUE THREAD

**Issue by chimpy** [home-assistant/homebridge-homeassistant#216]
Hey guys,

Getting the error mentioned in the title when I try to connect to HB.
I was using 2.1.0 with no problems.
Updated to 2.3.0 via sudo npm install -g homebridge-homeassistant@latest
Restarted HB, and it didn't appear in Home app when I tried to Add Accessory.
Changed the MAC address and PIN code a little in the config, restarted.
I now see HB when I click Add Accessory but after a few seconds, I get the error, "Home couldn't connect to this accessory."

Any ideas? :(

Thanks!

**Comment by jamieshaw** [comment]
@chimpy Have you tried deleting the 'Persist' folder in your configuration directory? That can sometimes alleviate any issues. The only downside is that you'll have to reconfigure anything in Homebridge that you may already have configured.

**Comment by chimpy** [comment]
@jamieshaw I did that, and also deleted my Home from the Home app. In the end there seems to bug that prevents me from adding "automation" to the support devices. When I do that, I cannot add Homebridge.

@robbiet480 Seems there is a bug with the automations? When I add "automation" to the supported types, I can no longer add Homebridge as an accessory, it fails.

**Comment by jjanderson** [comment]
Hi, I am seeing Home assistant over home-bridge, but I get an error saying "Couldn't add Home Assistant - Home couldn't connect to this accessory.. Any tips/tricks/advice? Thanks

**Comment by hvbhome** [comment]
yep, still the same here, after trying i believe to be all options:...

there's 2 ways i can connect the add-on to my Hassio install:

-  Either through the http://hassio/homeassistant setup, no port specified, no pw. Local in the add-on setting.

I can see the log connect to all devices, and give me the instruction to ener the 8 figure code. After that nothing happens. No more log entries, and no connection in the App.

- the other way is by means of my https://xxx.duckdns:8123 address, with pw specified. 
Completely empty settings in the Hassio except for the logging

I get the same log setup, including the code instruction. 
Even better, i can see all device-events registered  in the log. An enormous amount of things happening .... 

So looks promising, apparently the combination of Hassio and the Add-on is successful.

Still: no possible way to connect the App. Checked it all (i think). All networks. Deleted the persist folder. retried another username. Deleted and reinstalled the App. restarted HomeAssistant...

Hope @frenck might have some other suggestions to try.
Really would love to talk to my Homeassistant....


cheers,
Marius

**Comment by jamieshaw** [comment]
@hvbhome Just checking; where is the process falling down?
Are you unable to connect your iOS device to the Homebridge "hub", or just that after connecting the two, no Home Assistant devices are being populated in HomeKit?

**Comment by hvbhome** [comment]
I can get up to the dialogue to add HomeAssistant in Home, enter the figures
, but that always errors out stating it can’t connect. 

In my homebridge logs I see all is well in HA side of things, and all state changes are seen and logged. 

There only a warning about the dns setup, but I’m not sure this is causing it.

**Comment by jdagerot** [comment]
I have exact this problem too. Homebridge shows up as an accessory with icon and all and when I click on the icon the iOS-app says that it's connection. After a minute or so it comes back with an error, can't connect.

I'm running the latest non-beta on everything, both hass.io, add-ons and iOS

**Comment by hvbhome** [comment]
@jdagerot 
hi,

all of a sudden it Home has connected to HomeAssistant in my setup, please see https://community.home-assistant.io/t/community-hass-io-add-on-homebridge/33803/138?u=mariusthvdb for the settings that work. (tbh, i had all this before, so no new info maybe, but it works for now)

Hope this helps,
Marius

**Comment by jdagerot** [comment]
Thank you for your reply, I headed over to community-forum and my setup is identical as yours, put I couldn't get it to work so I gave up and headed for the TV. Then I came to wonder if my devices are on the same net, they share WiFi and DHCP and all that, but the hass.io had connected on my 2.4Ghz net while my phone was on 5Ghz. Changing the phone to the 5 and everything worked out directly.

No I need to figure out how I can force the hassio to use my 5ghz network instead...
